style.css

```diff
@@ -0,0 +1,64 @@
+/* * {
+    margin: 0;
+    padding: 0;
+} */
+
+#title {
+  text-align: center;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: 30px;
+  text-decoration: underline;
+}
+
+main p {
+  text-align: center;
+}
+
+main div p {
+  font-size: 10px;
+  text-align: justify;
+  max-width: 70%;
+  margin-top: 10px;
+  margin-bottom: 30px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
```

Comment:
I would like to advise you to consider comment every section of your style sheet, so everybody can identify the style of every part of the project.